src/web/app/pages-instructor/instructor-session-edit-page/instructor-session-edit-page.component.ts

```diff
@@ -400,9 +409,7 @@ export class InstructorSessionEditPageComponent extends InstructorSessionBasePag
    * Handles editing existing session event.
    */
   editExistingSessionHandler(): void {
-    this.sessionEditFormModel.isEditable = false;
```

Comment:
This is moved under `updateFeedbackSession`, when it's actually doing the updating. Otherwise to toggle it back, it'll be at the catch function in the modal, and also in the `updateFeedbackSession`, which might get pretty messy